tests/integration/test_cubepart.py

```diff
@@ -402,6 +402,36 @@ def and_it_respects_explicit_order_transform_for_measures(
         expected = load_python_expression(expectation)
         np.testing.assert_almost_equal(actual, expected)
 
+    @pytest.mark.xfail(reason="WIP", raises=NotImplementedError, strict=True)
+    def it_can_sort_by_column_percent(self):
+        """Responds to order:opposing_element sort-by-value.
+
+        So far, this is limited to column-percents (column-proportions) measure, but
+        others will follow.
+        """
+        transforms = {
+            "rows_dimension": {
+                "order": {
+                    "type": "opposing_element",
+                    "element_id": 1,
+                    "measure": "col_percent",
+                    "direction": "ascending",
+                    # --- element-ids are 1, 2, 3, 999 ---
+                    "exclude": {"top": [999], "bottom": [1]},
+                }
+            }
+        }
+        slice_ = _Slice(Cube(CR.CAT_4_X_CAT_5), 0, transforms, None, 0)
+
+        expected = [
```

Comment:
Maybe a non-sorted version would be useful for comparison (for the test reader) alongside this one.

Reply:
I don't care much for compound tests, but I can add some comments so a reader can tell which parts of the expectation show what aspect.